RxLocalizer.podspec

```diff
@@ -10,8 +10,8 @@ s.license = { :type => "MIT", :file => "LICENSE" }
 s.author = { "Vladislav Khambir" => "vlad.khambir@gmail.com" }
 s.homepage = "https://github.com/RxSwiftCommunity/RxLocalizer"
 s.source = { :git => "https://github.com/RxSwiftCommunity/RxLocalizer.git", :tag => "#{s.version}" }
-s.dependency 'RxSwift', '~> 6.2.0'
-s.dependency 'RxCocoa', '~> 6.2.0'
+s.dependency 'RxSwift', '~> 6.0'
+s.dependency 'RxCocoa', '~> 6.0'
```

Comment:
@ShenYj Why did you downgrade from 6.2 to 6.0?

Reply:
> @ShenYj Why did you downgrade from 6.2 to 6.0?

The previous version number was limited to the minor version, and will now be upgraded to 6.x

e.g. 6.5